README.md

```diff
@@ -35,6 +35,13 @@ To preview the website locally:
 
    Then, load <http://localhost:4000> in your browser.
 
+## Local Setup using Docker
+
+0. Ensure that you have Docker and Make installed.
+1. Run `make build` to build the project.
+2. Run `make serve` to serve the project, this include live reloading so any change will be reflected (it can take a while, check the logs).
+3. Run `make clean` to remove the docker images and resources generated.
```

Comment:
Perhaps you can also say that there is a make help command that lists this same help info.

Reply:
Thanks for the suggestion! Added in https://github.com/expressjs/expressjs.com/pull/1488/commits/d5aa944e964387b42d42163d28c21028e6a08cb8